feat: adding labs page (#114)
Basis-Theory/basistheory-integration-guides: collections/labs/labs.md

---
# Feel free to add content and custom Front Matter to this file.
# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults

layout: home
title: Basis Theory Labs
permalink: /labs
nav_order: 1
has_children: false
image:
    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
    width: 1200
    height: 630
---

<html>
    <head>
        <meta charset="utf-8">
        <title>Basis Theory Labs</title>
    </head>
    <body>
        <div class="header">
            <h1>Basis Theory Labs</h1>

>             <h5>Sample apps, demos and hacks powered by Basis Theory.</h5>

Reviewer: hacks?

            <p class="header-description">

>             Basis Theory enables you to tokenize anything, anywhere. Here you can get a sneak peek of Basis Theory's potential, by looking at some interesting usages we built for you! Securing data is a must, and whether it is on infrastructure as code, secret sharing, content management systems or anything you can think about, we got you covered. 

Reviewer: ```suggestion
            Basis Theory enables you to tokenize anything, anywhere. Here you can get a sneak peek of Basis Theory's potential by looking at some interesting usages we built for you! Securing data is a must, and whether it is in the context of infrastructure as code, secret sharing, content management systems or anything else you can imagine, we have you covered. 
```
